src/routes/backoffice.ts

```diff
@@ -738,6 +739,7 @@ export const createReservationHandler = async (req, res) => {
     posEntryMode,
     description,
     senderIBAN,
+    merchantId,
```

Comment:
Find out if it's necessary to also update the `createCreditPresentment` method

Reply:
It's not ... we ignore that transaction type